conda_env_linux64.yml

```diff
@@ -34,3 +34,4 @@ dependencies:
 - vine
 - pip:
   - modestimage==0.1
+  - Deprecated==1.2.0
```

Comment:
make sure to update the anaconda build file with the new package:

https://github.com/oceanobservatories/ooi-config/blob/master/anaconda_build/mi-instrument/meta.yaml

coordinate with Tim so that he can help build and test the changes

Reply:
Done.